scripts/test-npm-package.js

```
@@ -0,0 +1,160 @@
+#!/usr/bin/env node
+
+/**
+ * Test the npm package locally
+ * This script helps validate the npm package before publishing
+ */
+
+const { execSync, spawn } = require('child_process');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)